CI/physmon/physmon.py

```diff
@@ -175,21 +177,27 @@
             trackingGeometry,
             field,
             CKFPerformanceConfig(ptMin=400.0 * u.MeV, nMeasurementsMin=6),
+            TrackSelectorRanges(
+                removeNeutral=True,
+                absEta=(None, 2.5),
+                loc0=(None, 4.0 * u.mm),
+                pt=(500 * u.MeV, None),
+            ),
             outputDirRoot=tp,
             outputDirCsv=None,
         )
 
+        addAmbiguityResolution(
+            s,
+            AmbiguityResolutionConfig(
+                maximumSharedHits=3,
+            ),
+        )
+
         addVertexFitting(
             s,
             field,
-            TrackSelectorRanges(
-                removeNeutral=True,
-                absEta=(None, 2.5),
-                loc0=(None, 4.0 * u.mm),
-                pt=(500 * u.MeV, None),
-            ),
             vertexFinder=VertexFinder.Iterative,
-            trajectories="trajectories",
```

Comment:
Does it still run the correct truth matching in that case?

Reply:
hmm looking at this now I am not sure it ever did because I though it would only do that if `trajectories` are not present. the param is defaulted so the code should be doing the same thing